src/main/java/study2/domain/Players.java

```diff
@@ -0,0 +1,45 @@
+package study2.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.platform.commons.util.StringUtils;
+
+public class Players {
+
+	private static final String empty = " ";
+	List<Player> players;
+	
+	public Players(List<String> names) {
+		this.players = createPlayers(names);
+	}
+
+	private List<Player> createPlayers(List<String> names) {
+		
+		namesLimiation(names);
+		
+		return names.stream()
+				.map(Player::new)
+				.collect(Collectors.toList());
+	}
+
+	private void namesLimiation(List<String> names) {
+		if("".equals(names) || names == null ) {
+			throw new IllegalArgumentException("최소한 1명 이상이 필요합니다.");
+		}
+		
+	}
+
+	public int getPlayersCount() {
+		return this.players.size();
+	}
+	
+	public List<Player> getPlayers() {
+		
+		return players;
```

Comment:
players라는 Collection의 참조를 그대로 노출하고 있네요, 이런 경우 List<Player>의 변경이 일어날 수 있습니다.
Stream이나 `Collections.unmodifiablelist`를 사용해 불변을 보장해주시는게 좋습니다.

```suggestion
    return Collections.unmodifiablelist(players);
```

Reply:
저 콜렉션을 이용하면 외부에서 변경이 불가능 하여서 안전성을 높여 준다는 말씀이신가요?

Reply:
OCP (개방폐쇄의 원칙: Open Close Principle)
소프트웨어의 구성요소(컴포넌트, 클래스, 모듈, 함수)는 확장에는 열려있고, 변경에는 닫혀있어야 한다.

이부분과 관련있는건가요?